### PR TITLE
MEN-3987 mender-artifact tests: sudo usage on/off if present/absent.

### PR DESCRIPTION
**Pull request by merlin-northern**
ChangeLog:none
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

**Comment by merlin-northern** [comment]
> Hello 😸 I created a pipeline for you here: [Pipeline-202296100](https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/202296100)
> 
> Build Configuration Matrix

this pipeline should fail as https://github.com/mendersoftware/mender-artifact/pull/305 is not yet merged.

**Comment by merlin-northern** [comment]
> Hello 😸 I created a pipeline for you here: [Pipeline-202376735](https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/202376735)
> 
> Build Configuration Matrix

this pipeline should fail as mendersoftware/mender-artifact#305 is not yet merged.

**Comment by merlin-northern** [comment]
pipeline: https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/202444007

**Comment by merlin-northern** [comment]
> pipeline: https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/203740811

all passed

File: meta-mender-qemu/recipes-extended/sudo/sudo_%.bbappend [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FILES_${PN} += "/data/usr"
+do_install_append_qemux86-64 () {
+ mkdir -p ${D}/data/usr/bin
+ mv ${D}/usr/bin/sudo ${D}/data/usr/bin/
+ ln -s /data/usr/bin/sudo ${D}/usr/bin/sudo
```
Comment (kacf) [inline meta-mender-qemu/recipes-extended/sudo/sudo_%.bbappend:5]: Standard indentation in meta-mender is four spaces.

**Review by kacf — CHANGES_REQUESTED** [review]
One nitpick, looks good apart from that!

**Review by kacf — APPROVED** [review]

File: meta-mender-qemu/recipes-extended/sudo/sudo_%.bbappend (hunk 1 above)
Comment (merlin-northern) [reply meta-mender-qemu/recipes-extended/sudo/sudo_%.bbappend]: right! thanks!

**Pull request merged**